includes/gateway/class-omise-payment-alipay.php

```diff
@@ -68,13 +68,20 @@ public function charge( $order_id, $order ) {
 				apply_filters( 'omise_charge_params_metadata', array(), $order ),
 				array( 'order_id' => $order_id ) // override order_id as a reference for webhook handlers.
 			);
+			$return_uri = add_query_arg(
```

Comment:
Feels like maybe we should have a method for building the `return_uri` in `Omise_Payment` class

Reply:
@guzzilar consider comment above.. I am approving, not a big deal and it is urgent.
tested manually according to your testing description, works fine.

Reply:
@jonrandy not sure when but I may reinvent the way to handle these callbacks later on after Bill Payment (or after TrueMoney Walltet).
To me, there are some parts that the code is redundant, moving out only `return_uri` without designing the whole rest won't be that so effective.

Reply:
Yep, I'd noticed other functionality that could move to the parent class too, but was trying to limit the scope of my comments to the changes in this PR

Reply:
ic. noted and thanks 👍